integration-tests/helpers.js

```diff
@@ -204,13 +203,24 @@ async function createSandbox (dependencies = [], isGitRepo = false,
   // We might use NODE_OPTIONS to init the tracer. We don't want this to affect this operations
   const { NODE_OPTIONS, ...restOfEnv } = process.env
 
-  await mkdir(folder)
+  await fs.mkdir(folder)
   await exec(`yarn pack --filename ${out}`) // TODO: cache this
   await exec(`yarn add ${allDependencies.join(' ')}`, { cwd: folder, env: restOfEnv })
 
+  const winVolumes = new Set()
   for (const path of integrationTestsPaths) {
-    await exec(`cp -R ${path} ${folder}`)
-    await exec(`sync ${folder}`)
+    if (process.platform === 'win32') {
+      await exec(`Copy-Item -Recurse -Path "${path}" -Destination "${folder}"`, { shell: 'powershell.exe' })
+      winVolumes.add(folder[0])
+    } else {
+      await exec(`cp -R ${path} ${folder}`)
+      await exec(`sync ${folder}`)
+    }
+  }
+  for (const winVolume of winVolumes) {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏ `folder` does not change, hence you should have a single volume. You could also do the `sync` for other platforms here.